lib/chef/resource/windows_certificate.rb

```diff
@@ -134,7 +138,7 @@ def add_cert(cert_obj)
 
         def add_pfx_cert
           store = ::Win32::Certstore.open(new_resource.store_name)
-          store.add_pfx(new_resource.source, new_resource.pfx_password)
+          store.add_pfx(new_resource.source, new_resource.pfx_password, new_resource.pfx_properties)
```

Comment:
Since this requires the newer gem we'll also want to bump the dependency to require the version that allows passing in pfx properties